### ISSUE TITLE
Confirmation Emails

### ISSUE THREAD

**Issue by jmt4** [thethomp/lammergeyer#15]
When someone registers an email, we want them to verify ownership. This task is slightly premature since the accounts app isn't finished yet but I'm nearly done with user creation (hopefully today). And then we can tackle this.

Not sure how to do this, but an idea is to send them an email which contains a code in the url header. When they click on it, we activate the user if the header has the correct info.

**Comment by thethomp** [comment]
Yeah, we should do something like this. It's a pretty common problem; I believe there are some libraries out there that help with this kind of thing, we should research a bit.

**Comment by jmt4** [comment]
Here is a package which supposedly takes care of this kind of thing:(https://pypi.python.org/pypi/django-email-confirm-la/0.2.2). Based on the documentation this looks promising. I'll explore it a bit soon.

**Comment by jmt4** [comment]
Addressed with https://github.com/thethomp/lammergeyer/pull/31

There is still quite a bit more work that needs to be done on this part. As far as tests go, this should work but we probably want to back this up by some manual testing and/or a functional test

Additionally, we should probably add a few more things to how this works. For example, the current set up is register->redirect to login page. We probably want to add some sort of message informing the user that a confirmation email has been sent. Also, we probably want to let them know the confirmation was successful when they click on the link. Currently, it does none of that (django-registration-redux provides part of this out of the box).

Lastly, the full URL is not given in the email activation link. This has something to do with django/contrib/sites, like we need to register our site on the database then the sites framework will grab it on load-up or something.

So for now maybe we leave this issue open.
